app/models/slot.rb

```diff
@@ -0,0 +1,4 @@
+class Slot < ApplicationRecord
+  belongs_to :user
+  has_one :appointment
```

Comment:
This association won't work as there is not table named "appointment"

Use proper name for associtions